<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleDeviceSettings</class>
 <widget class="QWidget" name="SingleDeviceSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="applyChanges">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>250</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply Changes</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="backupFrequency">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>110</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Backup Frequency (seconds):</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseFile">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Directory</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Files to omit:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
